docs/development/code-review-guidelines/README.md

```diff
@@ -1,56 +1,105 @@
 
-Most of our code does not yet adhere to this styleguide, but we want all new code to adhere to it. You do not have to improve existing code when making changes, but we encourage it. If you do, please do all improvements in a separate commit from the actual change, so the improvements do not hide your actual code changes in a diff.
+The same is true for eslint. Your editor will likely have support for eslint checks, and allows you to correct them before committing.
 
-Before committing, please run your new code through [Rubocop](https://github.com/bbatsov/rubocop). It detects deviations from a lot of things in the style guide and things that are bad practice in general. You obviously do not have to fix issues with existing code. There is a [list of editor plugins](https://docs.rubocop.org/rubocop/1.31/integration_with_other_tools.html#editor-integration) in the Rubocop docs.
 
-When reviewing code and you think the author has not run the code through Rubocop, please ask them to.
 
-## Commit messages
+### Structure of commit messages
 
 - First line: less than 72 characters, this is when GitHub shows ‘…’
 - Blank line
 - Detailed description of the change, wrapped to 72 characters so the text is readable in git log
 
 See the [Git Book](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines).
 
+
+
 ## Testing
 
-- All tests must be green on continuous integration
+- All GitHub workflow actions must be green on continuous integration
 - Appropriate unit and integration tests, e.g. test application logic via rspec tests and its integration with the user interface via Cucumber tests
-- Test JavaScript code, e.g. via Jasmine or Selenium/Cucumber, but: Cucumber tests that do not test JavaScript must not have the @javascript tag, otherwise they run slower
-- Look for sufficient coverage of both Ruby and JavaScript code. CI will provide at least Ruby line coverage information at some point in the future
+- Test Frontend code with appropriate feature specs.
 - Forms: besides testing for success response, also test whether values are actually saved, e.g. read form or look into database
 - Bugfixes: must contain a test which detects the bug they fix to prevent regressions
 - Translations: Never use a specific translation string in a test. We might want to change them in the future and do not want to fix tests when we do.
+- Manual testing: When testing a feature or change, check out the code test at least the happy paths according to the specification of the ticket.
```

Comment:
Maybe a note that we are aware of flickering specs, actively trying to remove them. Something along

> - We are aware that there are some flickering specs in our codebase, that might fail randomly. We are actively trying to fix those. If you encounter test failures in code that you have not touched, try re-running the specs.